docs/user/guides/setup/configuration-file.rst

```diff
@@ -0,0 +1,223 @@
+How to add a configuration file
+===============================
+
+As part of the initial setup for your Read the Docs site,
+you need to create a **configuration file** called ``.readthedocs.yaml``.
+The configuration file tells Read the Docs what specific settings to use for your project.
+
+This how-to guide covers:
+
+#. Where to put your configuration file.
+#. Templates to setup your configuration file from scratch.
+#. The most basic sections in the configuration file that can help you get started.
+#. Next steps...
+
+.. seealso::
+
+   :doc:`/tutorial/index`.
+     Following the steps in our tutorial will help you setup your first documentation project.
+
+
+Where to put your configuration file
+------------------------------------
+
+The ``.readthedocs.yaml`` file should be placed in the top-most directory of your project's repository.
+We will get to the contents of the file in the next steps.
+
+When you have changed the configuration file,
+you need to commit and push the changes to your Git repository.
+Read the Docs will then automatically find and use the configuration to build your project.
+
+.. note::
+
+    Why is it called .yaml? What is that?
+      The configuration file is a `YAML`_ file. YAML files are a "map": a collection of
+      key-value pairs that can be nested. This is not unlike a JSON file or ``dict``
+      object in Python. This page won't explain the structure of YAML files, but many resources exist
+      online.
+
+
+.. _YAML: https://en.wikipedia.org/wiki/YAML
+
+.. _howto_templates:
+
+Getting started with a template
+-------------------------------
+
+Here are some variations of the configuration file that can help you get started.
+Pick the one that resembles your project the most,
```

Comment:
Some more exact language:

```suggestion
Pick an example based on the tool your project is using,
```

Reply:
I added: Pick an example based on the tool **that** your project is using